hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/RouterYarnClientUtils.java

```diff
@@ -126,50 +126,55 @@ private static ApplicationReport mergeUAMWithUAM(ApplicationReport uam1,
   }
 
   private static void mergeAMWithUAM(ApplicationReport am,
-      ApplicationReport uam){
+                                     ApplicationReport uam){
     ApplicationResourceUsageReport amResourceReport =
-        am.getApplicationResourceUsageReport();
+            am.getApplicationResourceUsageReport();
 
     ApplicationResourceUsageReport uamResourceReport =
-        uam.getApplicationResourceUsageReport();
+            uam.getApplicationResourceUsageReport();
 
-    amResourceReport.setNumUsedContainers(
-        amResourceReport.getNumUsedContainers() +
-            uamResourceReport.getNumUsedContainers());
+    if (amResourceReport == null) {
+      am.setApplicationResourceUsageReport(uamResourceReport);
+    } else if (uamResourceReport != null) {
 
-    amResourceReport.setNumReservedContainers(
-        amResourceReport.getNumReservedContainers() +
-            uamResourceReport.getNumReservedContainers());
+      amResourceReport.setNumUsedContainers(
+              amResourceReport.getNumUsedContainers() +
```

Comment:
Fix indentation as it was in the old one.

Reply:
Applied Hadoop code style to fix indentations.